sites/game-client/input-manager.mjs

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: BSD-2-Clause
 
 const KEYSTROKE_RATE = 30; // keystroke per second
+const CANVAS_AUTO_FOCUS_MS = 200;
```

Comment:
I can't be certain that it doesn't trigger reflow. Maybe double check that it doesn't or I would want to be conservative and put this value at 2000ms or 5000ms to avoid jitter?

Reply:
Can't see any possible code that would trigger a [browser reflow](https://gist.github.com/paulirish/5d52fb081b3570c81e3a).